### PR TITLE
Issue#646 hide save message in DiD Mode

### PR DESCRIPTION
**Pull request by RinneT**
This commit hides Game Saved messages in Dead is Dead mode to prevent message spamming

### CODE REVIEW

**Comment by lynxlynxlynx** [comment]
same thing, but we can just squash merge it.

**Comment by RinneT** [comment]
Sorry, I'm quite rusty with git and haven't used it in a while.
Also I just got what you meant.
I have another PR ready for #650 and would close this one to provide a clean one.
However I still get the many commits from my merges with upstream. rebase and squash ignore them for my repo. Any help how I can clean them?

**Comment by lynxlynxlynx** [comment]
git branch fix646
git checkout fix646
git reset --hard commit-hash-of-the-646-fix
git rebase origin # if you kept origin pointing to our repo, not yours
git push fix646
then go to github and you'll see it knows you just pushed a new branch and offers you to make a PR.

git checkout master
git branch fix650
git checkout fix650
git reset --hard commit-hash-of-the-last-upstream-commit
git cherry-pick 7424700  # or whatever the hash of the fix commit is
git push fix650
...

File: src/game/SaveLoadScreen.cc [hunk 1]
```diff
@@ -1692,8 +1692,9 @@ void DoDeadIsDeadSave()
 	// - we are in an AI Turn
 	// - we are in a Dialogue
 	// - we are in Meanwhile.....
+    // - we are in a locked ui
```
Comment (lynxlynxlynx) [inline src/game/SaveLoadScreen.cc]: spaces are banned, Deidranna would have you shot!

**Comment by lynxlynxlynx** [comment]
pushing more commits to this branch will just make more work for you. In situations like this, you can use git commit --amend to fix the previous commit and then force push (-f) to get it back up.

**Comment by RinneT** [comment]
Thanks for the detailed instructions. Took some notes for the future.
It worked until the push. I have origin set to my own repo. Changed it to yours and got:

> fatal: 'fix646' does not appear to be a git repository
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.

Trying with git push -u origin fix646 gave me:
>ERROR: Permission to ja2-stracciatella/ja2-stracciatella.git denied to RinneT.

**Comment by Manuel-K** [comment]
> Add more commits by pushing to the master branch on RinneT/ja2-stracciatella.

This PR is associated with your master branch.

**Comment by lynxlynxlynx** [comment]
Yeah, so "git push origin fix646" should do it, if origin still points to yours (RinneT/ja2-stracciatella).

But first to update with us. Instead of that rebase, do something like:
git fetch upstream && git rebase upstream/master # where upstream is us

**Comment by RinneT** [comment]
Thanks for the help again. Did a couple of things but think my repo is really messed up. I'll close this pr again and see if I can figure it out.

**Comment by Manuel-K** [comment]
You've included a lot of hidden files (e.g. `.directory`, `.kdev4/_custom.kdev4`) and compiler generated files (e.g. `*.d` and `*.o`).

**Comment by Manuel-K** [comment]
I think the easiest way to fix things is to copy the changed files out of the tree and hard reset everything.

**Pull request closed without merging**